tools/deb/envoy_bootstrap_v2.json

```diff
@@ -163,58 +166,100 @@
         ]
       }
       {{ end }}
+      {{ if .lightstep }}
+      ,
+      {
+        "name": "lightstep",
+        "http2_protocol_options": {},
+        {{ if .lightstepSecure }}
+        "tls_context": {},
```

Comment:
Open question: How can we provide the ca_cert that was used for signing the certs of the LightStep Satellites. From the code that I've read, the Citadel generated certs get mounted into the proxy container but the Citadel CA is not the one signing the LightStep Satellites. I saw some docs around using your own ca-cert (https://istio.io/docs/tasks/security/plugin-ca-cert/) which might work in some cases but I feel like there should be a more flexible mechanism though I don't know what it is. Docs on this envoy config option are here for reference (https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/auth/cert.proto#envoy-api-msg-auth-upstreamtlscontext)